front/app/containers/Admin/formBuilder/components/FormBuilderSettings/ContentSettings/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+// components
+import { Box, Text, colors } from '@citizenlab/cl2-component-library';
```

Comment:
Just moved all the regular settings to a new component to clean up the parent "FormBuilderSettings" component.